tensorflow/python/data/experimental/ops/data_service_ops.py

```diff
@@ -318,35 +318,48 @@ def distribute(processing_mode,
   a "one_epoch" mode which partitions the dataset across the tf.data
   workers, so that the consumers see each element of the dataset only once.
 
-  ```
-  dataset = tf.data.Dataset.range(5)
-  dataset = dataset.map(lambda x: x*x)
-  dataset = dataset.apply(
-      tf.data.experimental.service.distribute("parallel_epochs",
-                                              "grpc://dataservice:5000"))
-  dataset = dataset.map(lambda x: x+1)
-
-  for element in dataset:
-    print(element)  # prints { 1, 2, 5, 10, 17 }
-  ```
+  To see the distributed operations in action, the `DispatchServer` should be
+  started first so that tf.data workers can register to it.
 
-  In the above example, the first two lines (before the call to `distribute`)
-  will be executed on tf.data workers, and the elements provided over
-  RPC. The remaining transformations (after the call to `distribute`) will be
-  executed locally.
+  >>> dispatcher = tf.data.experimental.service.DispatchServer()
+  >>> print(dispatcher.target) # prints grpc://localhost:<port>
```

Comment:
This might break the doctest since we don't know what output to expect. I suggest just removing this line.

Reply:
addressed in the commit.